presto-main/src/main/java/com/facebook/presto/operator/WorkProcessorUtils.java

```diff
@@ -69,9 +69,9 @@ protected T computeNext()
             extends AbstractIterator<Optional<T>>
     {
         @Nullable
-        private WorkProcessor<T> processor;
+        WorkProcessor<T> processor;
```

Comment:
oh no! they were redundant -- _technically_ -- but... 😢

Reply:
I thought you don't like meaningless `private` in nested classes